### PR TITLE
[Doc Improvement][Document localization schema v1.17]

### PR DESCRIPTION
**Pull request by samantharamon**
Documents [v1.17](https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.Localization.schema.json) of the localization schema.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/937a64a1b8c10ed3f4188269b5653f2231627cd4/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a813b4f-6812-4ca5-8775-e31e57256dd7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/localization-schema.md [hunk 1]
```diff
@@ -55,10 +55,81 @@ The schema defines the following properties:
 |`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.taskInfo\\.title`|String|64|Replaces the corresponding strings from the app manifest with the value provided here.||
 |`activities.activityTypes\\[\\b([0-9]|[1-8][0-9]|9[0-9]|1[01][0-9]|12[0-7])\\b]\\.description`|String|128|A brief description of the notification.||
 |`activities.activityTypes\\[\\b([0-9]|[1-8][0-9]|9[0-9]|1[01][0-9]|12[0-7])\\b]\\.templateText`|String|128|Ex: "{actor} created task {taskId} for you".||
-|`meetingExtensionDefinition.scenes \\[[0-9]\\]\\.name$`|String|128|Replaces the corresponding strings from the app manifest. |
+|`meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name`|String|128|Replaces the corresponding strings from the app manifest. ||
+|`extensions\\[[0]\\]\\.audienceClaimUrl$`|String|2048|Replaces the corresponding string from the app manifest with the value provided here.||
```
Comment (v-ypalikila) [inline msteams-platform/resources/schema/localization-schema.md:59]:
```suggestion
|`extensions\\[[0]\\]\\.audienceClaimUrl$`|String|2048|Replaces the corresponding strings from the app manifest with the value provided here.||
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8c9951d53acb310e95321ece6400668eebce3fc8/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c6dc813-f621-424a-a68e-5f698678b7ea) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/localization-schema.md [hunk 2]
```diff
@@ -55,10 +55,81 @@ The schema defines the following properties:
 |`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.taskInfo\\.title`|String|64|Replaces the corresponding strings from the app manifest with the value provided here.||
 |`activities.activityTypes\\[\\b([0-9]|[1-8][0-9]|9[0-9]|1[01][0-9]|12[0-7])\\b]\\.description`|String|128|A brief description of the notification.||
 |`activities.activityTypes\\[\\b([0-9]|[1-8][0-9]|9[0-9]|1[01][0-9]|12[0-7])\\b]\\.templateText`|String|128|Ex: "{actor} created task {taskId} for you".||
-|`meetingExtensionDefinition.scenes \\[[0-9]\\]\\.name$`|String|128|Replaces the corresponding strings from the app manifest. |
+|`meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name`|String|128|Replaces the corresponding strings from the app manifest. ||
```
Comment (erikadoyle) [inline msteams-platform/resources/schema/localization-schema.md:58]:
```suggestion
|`meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name$`|String|128|Replaces the corresponding strings from the app manifest. ||
```

File: msteams-platform/resources/schema/localization-schema.md [hunk 3]
```diff
@@ -55,10 +55,81 @@ The schema defines the following properties:
...
+|`extensions\\[[0]\\]\\.audienceClaimUrl$`|String|2048|Replaces the corresponding strings from the app manifest with the value provided here.||
+|`extensions\\[[0]\\]\\.ribbons\\[[0-9]\\]\\.tabs\\[[1]?[0-9]\\]\\.label`|String|64|Replaces the corresponding strings from the app manifest with the value provided here.||
```
Comment (erikadoyle) [inline msteams-platform/resources/schema/localization-schema.md:60]:
```suggestion
|`extensions\\[[0]\\]\\.ribbons\\[[0-9]\\]\\.tabs\\[[1]?[0-9]\\]\\.label$`|String|64|Replaces the corresponding strings from the app manifest with the value provided here.||
```

**Review by erikadoyle — APPROVED** [review]
I checked this against the published v1.17 loc schema; looks good!

**Comment by erikadoyle** [comment]
@surbhigupta12 as this is simply a reflection of the already approved/published v1.17 loc schema, could we move forward with publishing? (I verified that it matches the schema XPaths and limits.) Thanks!

File: msteams-platform/resources/schema/localization-schema.md [hunk 4]
```diff
@@ -8,18 +8,18 @@ ms.date: 05/20/2019
...
+The Microsoft Teams localization file describes language translations that are served based on the client language settings. Your file must conform to the schema hosted at [`https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.Localization.schema.json`](https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.Localization.schema.json).
 
 > [!TIP]
-> Specify the schema at the beginning of your manifest to enable `IntelliSense` or similar support from your code editor: `"$schema": "<https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json>".
+> Specify the schema at the beginning of your manifest to enable `IntelliSense` or similar support from your code editor: `"$schema": "<https://developer.microsoft.com/json-schemas/teams/v1.17/MicrosoftTeams.schema.json>".
 
 ## Example
```
Comment (Wajeed-msft) [inline msteams-platform/resources/schema/localization-schema.md:16]:
```suggestion
## Localization schema v1.17
```

Could you please change this? Looking at "In this Article" section, developer might just navigate to 1.16 schema version. 
image

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/31623e7a9367c6b3ddd9aaaa6918154ed9aac14e/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/adcaf026-bb25-454b-b59e-82d9568fd4de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — DISMISSED** [review]

**Review by v-npaladugu — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90255da47187e99b3aed8eb690f3e550fed287ab/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e3c2f5c-c885-49df-ac23-0bace5731cb9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/98164f07686bf844bfdcb1ded0bad478f68b9d5c/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2a68ecc2-e0f5-417b-88f0-084351bed27e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/833766dc27f7b903a044e7e21b0e81934f9409da/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c197b46-5122-4d41-be7f-3fa95d28f550) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**